src/Entities/Entity.cpp

```diff
@@ -2372,12 +2364,22 @@ void cEntity::BroadcastDeathMessage(TakeDamageInfo & a_TDI)
 		{
 			GetWorld()->BroadcastChatDeath(DeathMessage);
 		}
-	}
-	else
+	}  // This will trigger if a player / tamed pet has been killed by another mob / tamed pet.
+	else if (a_TDI.Attacker->IsMob())
```

Comment:
Should we handle non-mob death attributions - is such a thing possible?

Reply:
Can you given an example?

Reply:
I was just trying to make the point that the previous code handled the case where !Attacker->IsMob(), and this code doesn't. Should we?

Reply:
I'm not sure if there is anything left. Dame from no one is handled, damage from player is handled and damage from mobs. I don't know of any case left